src/params/travels.jl

```diff
@@ -0,0 +1,29 @@
+Base.@kwdef struct Holiday <: InfectionTravels
+    days::Float64 = 0
+    peak::Float64 = 0
+    height::Float64 = 0
+    minimum::Float64 = 0
+  end
+  
+  function (f::Holiday)(state::SimState, params::SimParams, event::Event, infection_time::Real=0.0)
+    @assert kind(event) == OutsideTransmissionEvent
+    ck = contactkind(event)
+    #if ConstantKernelContact !== ck && SporadicContact !== ck && FriendshipContact !== ck
+    #  return true # do not affect other types of contact than "outer" ones
+    #end
+    #rand(state.rng) < max((Float64(-abs(infection_time-f.peak)*abs(infection_time-f.peak)))/1500.0 + f.height,f.minimum)
+    vaule = rand(state.rng)
+    #probability =  max((Float64(-abs(infection_time-f.peak)*abs(infection_time-f.peak)))/1500.0 + f.height,f.minimum)
+    probability =  max((Float64(-abs(event.time-f.peak)*abs(event.time-f.peak)))/1500.0 + f.height,f.minimum)
```

Comment:
can you explain how does this function work? what does 1500 stand for?

Reply:
This is an inverted square function whose "peak" can be shifted to choose when you want the highest activity. Additionally, you can define the lower value of the function. 1500 was in the preliminary version. In the correct version there is a "days" factor which determines the length of travel activity.